### ISSUE TITLE
Focus issue when clicking outside chart - high contrast

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#1109]
**Please complete the following information** 

Open SDG version: 1.3.0-dev

**Describe the bug**

In high contrast mode, clicking around the chart (like in the chart footer) gives that div focus which makes it look completely yellow.

**To Reproduce**
1. Any indicator page
2. Switch to high contrast mode
3. Click on anything in the chart footer (like "Geographical area")

**Expected behavior**

Ideally there would be no focus needed there, but it might be necessary for the screen-reader. At least it should only be a yellow outline, and not completely filled yellow.

**Screenshots**

![chart-outline-focus-yellow](https://user-images.githubusercontent.com/1319083/108996221-1e30eb00-766c-11eb-8220-f0052b980c74.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version latest
